content/zh/docs/_index.md

```diff
@@ -5,13 +5,13 @@ description: "您需要知道的关于文档如何组织的所有内容。"
 
 # 欢迎
 
-欢迎使用 [Helm](https://helm.sh/) 文档。 Helm 是 Kubernetes 的包管理器，您也可以在 [CNCF Helm 项目过程报告](https://www.cncf.io/cncf-helm-project-journey/)阅读详细的背景信息。
+欢迎使用 [Helm](https://helm.sh/zh/) 文档。 Helm 是 Kubernetes 的包管理器，您也可以在 [CNCF Helm 项目过程报告](https://www.cncf.io/cncf-helm-project-journey/)阅读详细的背景信息。
 
 # 文档构成
 
 Helm 有大量的文档。高级组织概述会让您知道在哪里查找特定内容。
 
-- [教程](https://helm.sh/docs/intro) 如果您是新手，从这里开始，手把手带您通过一系列的步骤创建您的第一个 Helm 图表。
-- [主题引导](https://helm.sh/docs/topics) 以相当高的水平讨论关键主题和概念并提供有用的背景信息和解释。
-- [社区引导](https://helm.sh/docs/community) 讨论围绕社区的相关话题。如果您想了解更多关于 Helm 本身的开发过程以及如何参与开发，从这里开始。
-- [如何做](https://helm.sh/docs/howto) 操作方法。 指引您按照步骤参与定位关键问题和使用案例。假定您对 Helm 工作原理有所了解的情况下，这些内容会比教程更加高级。
+- [教程](https://helm.sh/zh/docs/intro) 如果您是新手，从这里开始，手把手带您通过一系列的步骤创建您的第一个 Helm 图表。
+- [主题引导](https://helm.sh/zh/docs/topics) 以相当高的水平讨论关键主题和概念并提供有用的背景信息和解释。
+- [社区引导](https://helm.sh/zh/docs/community) 讨论围绕社区的相关话题。如果您想了解更多关于 Helm 本身的开发过程以及如何参与开发，从这里开始。
+- [如何做](https://helm.sh/zh/docs/howto) 操作方法。 指引您按照步骤参与定位关键问题和使用案例。假定您对 Helm 工作原理有所了解的情况下，这些内容会比教程更加高级。
```

Comment:
Instead of adding `zh`, it's probably a better approach to do it like https://github.com/helm/helm-www/pull/813/files is doing - removing the domain and just using the relative path. Can you please change this? Thanks.

```
- [教程](intro) 指导您完成一系列步骤创建您的第一个 Helm Chart。
  如果您不熟悉Helm，请从这里开始。
- [主题指引](topics) 在较高水平上讨论关键主题和概念，
  并提供有用的背景信息和说明。
```

Reply:
OK，I got it. I'll check other part later.